python/paddle/vision/models/vgg.py

```diff
@@ -96,7 +117,7 @@ def forward(self, x):
         return x
 
 
-def make_layers(cfg, batch_norm=False):
+def make_layers(cfg: dict[str, list], batch_norm: bool = False) -> Sequential:
```

Comment:
```suggestion
def make_layers(cfg: dict[str, list[Any]], batch_norm: bool = False) -> Sequential:
```

不允许不指定泛型参数，即便使用 `Any`

泛型就算用 Any 也要标注